ExperimentConfigSidecar/Models/CPUUsage.cs

```diff
@@ -0,0 +1,8 @@
+namespace ExperimentConfigSidecar.Models;
+
+/// <summary>
+/// Represents the CPU usage configuration.
+/// </summary>
+/// <param name="UsageDuration">The duration in milliseconds to use CPU (keep a thread spinning).</param>
+/// <param name="PauseDuration">The duration in milliseconds to pause between CPU usage.</param>
```

Comment:
This could be a bit clearer in my opinion.
As far as I understand, it cycles between using the CPU for "UsageDuration" milliseconds, and pausing for "PauseDuration"?
So its not a one time configuration but forever?

Reply:
yes, it's forever, I add a bit more documentation with 142df587e32f5a91f2c00f29772d802fb18ba2a9